### ISSUE TITLE
Error 404 Page not found

### ISSUE THREAD

**Issue by Shivangi2430** [joschan21/bookbuddy#4]
Hey, 
firstly I'm getting error in my ChatInput file. It is saying that -

Property 'isLoading' does not exist on type 'UseMutationResult<ReadableStream<Uint8Array> | null, Error, { id: string; text: string; isUserMessage: boolean; }, void>'.ts(2339) 

even I tried copying the same code from the repo but it is still the same. 

Secondly I'm getting the error 404 page not found every time, Even I tried fixing the isLoading error and I fixed it but still getting the same error 404. There is no other error in my any of the files just this. Please help! The code I used in ChatInput is same as it is in the repo!

**Comment by 1-alexkarter** [comment]
you need to use isPending instead of isLoading

**Comment by Shivangi2430** [comment]
> you need to use isPending instead of isLoading

Yes I have already done this. But even if all the errors are gone it is still saying error 404 page not found.
Even I matched all the files and everything is correct. 
I have used my own secret key and all. There is no error but still I'm not able to run the site!
